### DISCUSSION TITLE
Stop or delay server from receiving spawn packet

### DISCUSSION THREAD

**Discussion by caykey** [PrismarineJS/mineflayer#1604] [Q&A]
Hey there, 

I was wondering if there is any way to halt or stop the world loading process so that the server doesn't get the spawn notice.
If not in normal mineflayer, where would the server world loading be in the actual code?

If you have any suggestions how to halt this process, if only for a second that would be very appreciated!

**Comment by rom1504** [comment]
This is happening in node-minecraft-protocol ; you would need to delay the
login sequence
